python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -471,10 +494,11 @@ def __set__(self, obj, val):
         new_value = Descriptor.cleanValue(obj, val, _set_name)
 
         obj.setSchemaAttribute(_set_name, new_value)
+
         obj._setDirty()
 
-        if isinstance(val, Node):
-            val._setDirty()
+        # This is currently disabled due to bad interaction with the GangaDirac tests
+        #Descriptor.check_inheritance(obj, _set_name)
```

Comment:
We shouldn't be putting in requests to merge code which is commented out like this.